test/e2e-istio-tests.sh

```diff
@@ -3,21 +3,6 @@
 # This script runs e2e tests for Canary, B/G and A/B initialization, analysis and promotion
 # Prerequisites: Kubernetes Kind and Istio
 
-set -o errexit
```

Comment:
Can you please add back `set -o errexit` to all scripts

Reply:
Of course.

ps: I don't know if this new structure of tests makes sense. I'm happy to change this if you want.